python/py-pymc3/Portfile

```diff
@@ -48,6 +50,9 @@ if {${name} ne ${subport}} {
         depends_lib-append \
                         port:py${python.version}-contextvars \
                         port:py${python.version}-dataclasses
+
+        depends_lib-delete \
+                        port:py${python.version}-semver
```

Comment:
shouldn't this also remove `port:py${python.version}-cachetools`?

Reply:
I don't know exactly what to do; I just tested py36-pymc3 under Python 3.6 and it turns out that even the outdated PyMC 3.10.0 fails importing because, meanwhile, Arviz has changed (PyMC 3.10.0 expects some arviz.geweke that doesn't exist any more in arviz 0.11.2).
 I initially left port:py${python.version}-cachetools because, contrarily to port:py${python.version}-semver, its port still supports Python 3.6.
 Maybe the easiest is to drop entirely Python 3.6 from the py-pymc3 port...

Reply:
well, PyMC 3.1.10 has `arviz>=0.9.0` as a requirement so one would assume that the current version of `py-arviz` would be fine... but you say that isn't the case. I don't want to create another subport of `py-arviz` limiting that one the an earlier version for PY36 just to support PyMC3. So yes, I am fine with dropping the PY36 subport from `python.versions` and removing the specific block for PY36.

Reply:
Done, thanks.